server/pages/index.js

```diff
@@ -370,11 +371,18 @@ module.exports = function() {
 			}
 		} );
 
-	app.get( '/browsehappy', function( request, response ) {
-		response.status( 200 ).render( 'browsehappy.jade', {
-			urls: generateStaticUrls( request )
+	app.get( '/browsehappy', setUpRoute, function( req, res, next ) {
+		const wpcomRe = /^https?:\/\/[A-z_-]+\.wordpress\.com$/;
+		const primaryBlogUrl = get( req, 'context.user.primary_blog_url', '' );
+		const isWpcom = wpcomRe.test( primaryBlogUrl );
```

Comment:
As opposed to Jetpack, I guess? But why not even point to the Jetpack site's wp-admin in that case? (And What Would Atlas Do?) `dashboard.wordpress.com/wp-admin/` seems only mildly useful in terms of functionality, TBH... 😄